service-front/module/Application/src/Model/Service/AbstractEmailService.php

```diff
@@ -25,16 +25,17 @@ abstract class AbstractEmailService extends AbstractService
      * implementations may map these to appropriate rendering mechanisms (e.g. 
      * 3rd party identifiers (e.g. Notify template IDs)
      */
-    public const EMAIL_ACCOUNT_ACTIVATE                = 'email-account-activate';
-    public const EMAIL_FEEDBACK                        = 'email-feedback';
-    public const EMAIL_LPA_REGISTRATION                = 'email-lpa-registration';
-    public const EMAIL_LPA_REGISTRATION_WITH_PAYMENT   = 'email-lpa-registration-with-payment';
-    public const EMAIL_NEW_EMAIL_ADDRESS_NOTIFY        = 'email-new-email-address-notify';
-    public const EMAIL_NEW_EMAIL_ADDRESS_VERIFY        = 'email-new-email-address-verify';
-    public const EMAIL_PASSWORD_CHANGED                = 'email-password-changed';
-    public const EMAIL_PASSWORD_RESET                  = 'email-password-reset';
-    public const EMAIL_PASSWORD_RESET_NO_ACCOUNT       = 'email-password-reset-no-account';
-    public const EMAIL_ACCOUNT_DUPLICATION_WARNING     = 'email-account-duplication-warning';
+    public const EMAIL_ACCOUNT_ACTIVATE                      = 'email-account-activate';
+    public const EMAIL_FEEDBACK                              = 'email-feedback';
+    public const EMAIL_LPA_REGISTRATION_WITH_PAYMENT1        = 'email-lpa-registration-with-payment1';
```

Comment:
I'm not sure we need the numbered suffixes, as the names are distinct anyway. Alternatively, if you're trying to make clear they're a set, I'd call them `email-lpa-registration-1-with-payment`, `email-lpa-registration-2-with-cheque-payment` etc. to make the grouping clear.

Reply:
@townxelliot the suffixes I simply matched what Amy called the templates within Notify.  I could change these if you really like, and have no objection, but then they either wouldn't match what Amy did or she'd have to rename the templates in Notify.  How much do you care about this?  ;)

Reply:
Not enough to cause additional work.